website/addons/wiki/templates/edit.mako

```diff
@@ -219,7 +219,7 @@
   <div class="modal-dialog">
     <div class="modal-content">
       <div class="modal-header">
-        <h3 class="modal-title">The permissions for this page have changed</h3>
```

Comment:
ping @haoyuchen1992 @caneruguz Should modal titles be titlecased even if the title is a full sentence like this?